controlpanel/api/aws.py

```diff
@@ -293,6 +295,9 @@ def put(self, policy_document=None):
             if stmt.get('Resource')
         ]
 
+        return self.save_policy_document(policy_document)
+
+    def save_policy_document(self, policy_document):
```

Comment:
Considering we're within the context of `S3AccessPolicy` class we may even call this just "save" 🤷‍♂

Reply:
I don't want to have both `save` and `put`. This method is for the subclass to override.